app/helpers/admin_helper.rb

```diff
@@ -14,4 +14,21 @@ def display_name(group_name)
   def dom_safe_name(group_name)
     group_name.gsub(/\s+|\./, "_").gsub(/\A(\d)/, '_\1')
   end
+
+  def current?(request, group, group_name)
+    comparable = group[:children][0][:parent] ? group[:children][0][:parent].to_s : group_name.to_s
```

Comment:
Unless I'm misreading your intention, this should do the same:

```suggestion
    comparable = (group[:children][0][:parent] || group_name).to_s
```